src/tools/bin/tbg

```diff
@@ -199,17 +214,11 @@ fi
 
 eval set -- "$OPTS"
 
-# default parameters from environment variables (e.g. in a .profile file)
-#   -s | --submit
-submit_command=${TBG_SUBMIT:-""}
-#   -t | --tpl
-tooltpl_file=${TBG_TPLFILE:-""}
-
 # parser
 while true ; do
     case "$1" in
         -s|--submit)
-            submit_command="$2"
+            submit_command=${2:-$TBG_SUBMIT}
```

Comment:
`$TBG_SUBMIT` == empty but `-s` set:

check from [below](https://github.com/ax3l/picongpu/pull/2/files#diff-505f822097fa129ec6efa8c58b35c965R346) must be added here for that case.